packages/stems/src/components/Button/Button.module.css

```diff
@@ -59,15 +59,15 @@
 }
 
 .button .icon.left.noText {
-  margin: 0;
+  margin: 0 !important;
```

Comment:
Can't wait for the design updates to avoid things like this in the future

Reply:
right! it's so frustrating